From 1ee425c753bbd70916b5835b0e5086f633b2e542 Mon Sep 17 00:00:00 2001
From: Timo Herbrecher <t.herbrecher@gateware.de>
Date: Fri, 12 Oct 2018 11:39:01 +0200
Subject: [PATCH] tqmt104x_rcw: Don't use CD signal to detect mmc

TQMT104x modules have eMMC and SD-Card support but card-detect signal is
not handled when eMMC is selected. Skip support for card-detect and
write-protect signals because card availability could be handled
otherwise.

Signed-off-by: Timo Herbrecher <t.herbrecher@gateware.de>
---
 board/tqc/tqmt104x/tqmt1040_serdes66_rcw.cfg | 2 +-
 board/tqc/tqmt104x/tqmt1042_serdes86_rcw.cfg | 2 +-
 board/tqc/tqmt104x/tqmt1042_serdes88_rcw.cfg | 2 +-
 board/tqc/tqmt104x/tqmt1042_serdes8e_rcw.cfg | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/board/tqc/tqmt104x/tqmt1040_serdes66_rcw.cfg b/board/tqc/tqmt104x/tqmt1040_serdes66_rcw.cfg
index 405dad6..674e450 100644
--- a/board/tqc/tqmt104x/tqmt1040_serdes66_rcw.cfg
+++ b/board/tqc/tqmt104x/tqmt1040_serdes66_rcw.cfg
@@ -4,4 +4,4 @@ aa55aa55 010e0100
 0c10000e 0e000000 00000000 00000000
 66000002 80000001 ec027000 01000000
 00000000 00000000 00000000 0002a800
-00000000 01fe5005 00000000 00000000
+00000000 41fe5005 00000000 00000000
diff --git a/board/tqc/tqmt104x/tqmt1042_serdes86_rcw.cfg b/board/tqc/tqmt104x/tqmt1042_serdes86_rcw.cfg
index 38597ae..7734543 100644
--- a/board/tqc/tqmt104x/tqmt1042_serdes86_rcw.cfg
+++ b/board/tqc/tqmt104x/tqmt1042_serdes86_rcw.cfg
@@ -4,4 +4,4 @@ aa55aa55 010e0100
 0c10000e 0e000000 00000000 00000000
 86000000 80000001 ec027000 01000000
 00000000 00000000 00000000 0002a800
-00000000 01fe5005 00000000 00000000
+00000000 41fe5005 00000000 00000000
diff --git a/board/tqc/tqmt104x/tqmt1042_serdes88_rcw.cfg b/board/tqc/tqmt104x/tqmt1042_serdes88_rcw.cfg
index 3c92767..461aa7b 100644
--- a/board/tqc/tqmt104x/tqmt1042_serdes88_rcw.cfg
+++ b/board/tqc/tqmt104x/tqmt1042_serdes88_rcw.cfg
@@ -4,4 +4,4 @@ aa55aa55 010e0100
 0c10000e 0e000000 00000000 00000000
 88000000 80000001 ec027000 01000000
 00000000 00000000 00000000 0002a800
-00000000 01fe5005 00000000 00000000
+00000000 41fe5005 00000000 00000000
diff --git a/board/tqc/tqmt104x/tqmt1042_serdes8e_rcw.cfg b/board/tqc/tqmt104x/tqmt1042_serdes8e_rcw.cfg
index 37c1ea2..283bd8d 100644
--- a/board/tqc/tqmt104x/tqmt1042_serdes8e_rcw.cfg
+++ b/board/tqc/tqmt104x/tqmt1042_serdes8e_rcw.cfg
@@ -4,4 +4,4 @@ aa55aa55 010e0100
 0c10000e 0e000000 00000000 00000000
 8e000000 80000001 ec027000 01000000
 00000000 00000000 00000000 0002a800
-00000000 11fe5005 00000000 00000000
+00000000 51fe5005 00000000 00000000
-- 
1.9.1

